airflow/auth/managers/base_auth_manager.py

```diff
@@ -83,21 +83,29 @@ def get_api_endpoints(self) -> None | FlaskApi:
         """Return API endpoint(s) definition for the auth manager."""
         return None
 
-    @abstractmethod
     def get_user_name(self) -> str:
         """Return the username associated to the user in session."""
+        user = self.get_user()
+        if not user:
+            self.log.error("Calling 'get_user_name()' but the user is not signed in.")
+            raise AirflowException("The user must be signed in.")
+        return user.get_name()
```

Comment:
@vincbeck This looks like a breaking change since `get_name()` method was just defined in the BaseUser class. WDYT?

Reply:
No because the auth manager interface is not public interface YET. It will become in the near future but work in auth manager is still in progress. It is not documented anywhere how to build an auth manager or even how to use it so I assume nobody uses it